CLAUDE.md

```diff
@@ -0,0 +1,268 @@
+# Sentry Development Guide for Claude
+
+## Overview
+
+Sentry is a developer-first error tracking and performance monitoring platform. This repository contains the main Sentry application, which is a large-scale Django application with a React frontend.
+
+## Tech Stack
+
+### Backend
+
+- **Language**: Python 3.13+
+- **Framework**: Django 5.2+
+- **API**: Django REST Framework with drf-spectacular for OpenAPI docs
+- **Task Queue**: Celery 5.5.2
+- **Databases**: PostgreSQL (primary), Redis, ClickHouse (via Snuba)
+- **Message Queue**: Kafka, RabbitMQ
+- **Cloud Services**: Google Cloud Platform (Bigtable, Pub/Sub, Storage, KMS)
+
+### Frontend
+
+- **Language**: TypeScript
+- **Framework**: React 19.1.0
+- **Build Tool**: Rspack (Webpack alternative)
+- **State Management**: Reflux, React Query (TanStack Query)
+- **Styling**: Emotion (CSS-in-JS), Less
+- **Testing**: Jest, React Testing Library
+
+### Infrastructure
+
+- **Container**: Docker (via devservices)
+- **Package Management**: pnpm (Node.js), pip (Python)
+- **Node Version**: 22.11.0 (managed by Volta)
+
+## Project Structure
+
+```
+sentry/
+├── src/
+│   ├── sentry/           # Main Django application
+│   │   ├── api/          # REST API endpoints
+│   │   ├── models/       # Django models
+│   │   ├── tasks/        # Celery tasks
+│   │   ├── integrations/ # Third-party integrations
+│   │   ├── issues/       # Issue tracking logic
+│   │   └── web/          # Web views and middleware
+│   ├── sentry_plugins/   # Plugin system
+│   └── social_auth/      # Social authentication
+├── static/
+│   ├── app/              # React application
+│   │   ├── components/   # Reusable React components
+│   │   ├── views/        # Page components
+│   │   ├── stores/       # State management
+│   │   └── utils/        # Utility functions
+│   └── fonts/            # Font files
+├── tests/                # Test suite
+├── fixtures/             # Test fixtures
+├── devenv/               # Development environment config
+├── migrations/           # Database migrations
+└── config/               # Configuration files
+```
+
+## Key Commands
+
+### Development Setup
```

Comment:
I see `sentry devserver` is mentioned in the Debugging section, should that be moved up here?